### DISCUSSION TITLE
G code

### DISCUSSION THREAD

**Discussion by Salehdanaee** [fra589/grbl-Mega-5X#335] [General]
Hello
Thank you very much for your efforts
I wanted to see how can I do the design for 5 axis?
I didn't do anything...

grblgru doesn't have this ability?

**Comment by fra589** [comment]
Hi @Salehdanaee,  
   
I'm sorry, but I don't understand what you want...  
What do you mean by:
> I wanted to see how can I do the design for 5 axis?
> I didn't do anything...
> grblgru doesn't have this ability?

@++;  
Gauthier.
